### PR TITLE
feat: voice clone support

### PR DESCRIPTION
**Pull request by piotrpcharactr**
## Describe your changes

Added Voice Clone support

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

- [X] My code follows the style guidelines of this project and meets the industry quality standards
- [X] I have made corresponding changes to the documentation, `CHANGELOG.md` and `README.md`
- [X] I have updated the package version according to [semantic versioning](https://semver.org/)
- [X] I have provided meaningful commit messages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,30 +35,35 @@ tts_result = sdk.tts.convert(tts_voices[0]["id"], "Hello world")
 
 #### Clone & install the SDK locally
 ```bash
-$ git clone https://github.com/charactr-platform/charactr-api-sdk-python
-$ cd charactr-api-sdk-python
-$ python setup.py sdist
+git clone https://github.com/charactr-platform/charactr-api-sdk-python
+cd charactr-api-sdk-python
+python setup.py sdist
```
Comment (mkosecki) [inline README.md]: Actually I don't know why you generate distribution file, why not `pip install .` if this is about installation

File: charactr_api/sdk/clones.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            data=multipart_data,
+        )
+
+        if response.status_code != 200:
+            raise get_api_error(response)
+
+        try:
+            voice = json.loads(response.content)
+        except Exception as e:
+            raise Exception(e)
```
Comment (mkosecki) [inline charactr_api/sdk/clones.py:70]: A nie wiem czy może nie warto wywalać więcej info o błędzie jak już jest Exception razem z `traceback` info.

https://docs.python.org/3.10/library/traceback.html#traceback-examples

File: README.md (hunk 1 above)
Comment (mkosecki) [reply README.md]: or even sdist may stay but to install `pip install dist/*.tar.gz `

**Review by mkosecki — COMMENTED** [review]
Aas in comments

File: README.md (hunk 1 above)
Comment (piotrpcharactr) [reply README.md]: Right, before we had the pipeline deploy, we had to manually build and push it, so this was covered in the readme. I'll update it.

File: charactr_api/sdk/clones.py (hunk 2 above)
Comment (piotrpcharactr) [reply charactr_api/sdk/clones.py:70]: Good point, added a task to review & improve error handling for all errors in the sdk.

**Review by mkosecki — APPROVED** [review]

**Pull request merged**